mne_sandbox/preprocessing/_sns.py

```diff
@@ -64,6 +73,11 @@ def fit(self, raw, verbose=None):
         -------
         sns : Instance of SensorNoiseSuppression
             The modified instance.
+
+        Notes
+        -----
+        In the resulting operator, bad channels will be reconstructed by
+        using the good channels.
```

Comment:
Bad channels will be reconstructed using the good channels, but bad channels will not contribute to the good ones (i.e., the bad channel columns in the operator will be zero). This is the difference.